model-inference/decisionTree/experiments/data_processing.py

```diff
@@ -239,10 +241,10 @@ def create_tables(
         raise ValueError(f"{dataset} not supported")
 
     # Split dataset
-    train, test = train_test_split(df, random_state=77, test_size=datasetconfig["test"])
-    del df
-    gc.collect()
-
+    # df = shuffle(df, random_state=77)
```

Comment:
Hey Hong, do you mean to uncomment this? We would like the split to be random, right?

Reply:
Yes. We can discuss this point.  I think it is okay not to shuffle the data. The shuffling operation uses more memory, which may cause memory issue for large datasets.

Reply:
Will shuffling affect accuracy? We want to make the benchmark realistic by working with good quality models.
If it won't affect accuracy, e.g., if the training process will also shuffle the data, we can definitely remove the operator to improve the performance of our benchmark pipeline.